tasks/main.yml

```diff
@@ -41,3 +41,15 @@
   when: "telegraf_plugins_extra is defined and telegraf_plugins_extra is iterable"
   become: yes
   notify: "Restart Telegraf"
+
+- name: "Deploy script files for exec input plugin"
+  unarchive:
```

Comment:
http://docs.ansible.com/ansible/latest/modules/unarchive_module.html

maybe specify in ansible that we need a remote archive ?
i dont know if this module can handle flat files too